README.md

```diff
@@ -1,6 +1,8 @@
 # K-9 Mail
-[![Build Status](https://k9mail.ci.cloudbees.com/job/master/badge/icon)](https://k9mail.ci.cloudbees.com/job/master/)
-[![Join the chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/k9mail/k-9)
+[![Build status](https://img.shields.io/travis/k9mail/k-9.svg?style=flat-square)](https://travis-ci.org/k9mail/k-9)
```

Comment:
Travis CI only builds pull requests, not the master branch. This build status is 3 years out of date. Please remove.